src/GUniLPOracle.t.sol

```diff
@@ -56,6 +56,10 @@ interface OracleLike {
 contract GUniLPOracleTest is DSTest {
 
     function assertEqApprox(uint256 _a, uint256 _b, uint256 _tolerance) internal {
+        assertEqApprox(_a, _b, _tolerance, "");
+    }
+
+    function assertEqApprox(uint256 _a, uint256 _b, uint256 _tolerance, string memory err) internal {
```

Comment:
this does not seem to be used in the new tests. I don't see a test that uses the `err`.  If this is a standardization thing should we add it to the not Eq Approx too?  Should our use of the assert include error messages?

Reply:
Added.

Reply:
I added it for the `assertNotEqApprox` as well in case someone needs it.